services/resetPassword.js

```diff
@@ -0,0 +1,23 @@
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt');
+const userRepository = require('../repositories/user');
+const errors = require('../configuration/errors');
+
+async function resetPassword(token, newPassword, confirmPassword) {
+  const decodedToken = jwt.verify(token, process.env.JWT_SECRET);
```

Comment:
same as the comment above regarding moving this to env file

Reply:
do we have some kind of validation if token is invalid?

Reply:
![Screenshot_8](https://github.com/becarselman/ant-colony-pharaoh-backend/assets/101573891/b6c83281-d9c5-423b-b409-7d4333deb6e1)

we get this kind of message